docs/en/getting-started/get-started-stack.asciidoc

```diff
@@ -0,0 +1,871 @@
+----------------------------------------------------------------------
+
+endif::[]
+
+*rpm:*
+
+ifeval::["{release-state}"=="unreleased"]
+
+Version {stack-version} of {ls} has not yet been released.
+
+endif::[]
+
+ifeval::["{release-state}"!="unreleased"]
+
+["source","sh",subs="attributes,callouts"]
+----------------------------------------------------------------------
+curl -L -O https://artifacts.elastic.co/downloads/logstash/logstash-{logstash_version}.rpm
+sudo rpm -i logstash-{logstash_version}.rpm
+----------------------------------------------------------------------
+
+endif::[]
+
+*mac:*
+
+ifeval::["{release-state}"=="unreleased"]
+
+Version {stack-version} of {ls} has not yet been released.
+
+endif::[]
+
+ifeval::["{release-state}"!="unreleased"]
+
+["source","sh",subs="attributes,callouts"]
+----------------------------------------------------------------------
+curl -L -O https://artifacts.elastic.co/downloads/logstash/logstash-{logstash_version}.zip
+unzip logstash-{logstash_version}.zip
+----------------------------------------------------------------------
+
+endif::[]
+
+*win:*
+
+ifeval::["{release-state}"=="unreleased"]
+
+Version {stack-version} of {ls} has not yet been released.
+
+endif::[]
+
+ifeval::["{release-state}"!="unreleased"]
+
+. Download the {ls} {logstash_version} Windows zip file from the
+https://www.elastic.co/downloads/logstash[{ls} download] page.
+
+. Extract the contents of the zip file to a directory on your computer, for
+example, `C:\Program Files`.
+
+endif::[]
+
+For other operating systems, go to the
+https://www.elastic.co/downloads/logstash[{ls} download] page.
+
+To learn more about installing, configuring, and running {ls}, read the
+{logstash-ref}/index.html[{ls} Reference].
+
+// REVIEWERS: I've removed the info about updating the beats input plugin for
+// Logstah because this tutorial tells people to download the latest version
+// and in most cases the input plugin version will be recent...so I'm not sure
+// we need to cover that here.
+
+[float]
+==== Configure {metricbeat} to send events to {ls}
+
+{metricbeat} sends events to {es} by default. To send events to {ls}, modify the
+{metricbeat} configuration file, `metricbeat.yml`. You'll find this file under
+the {metricbeat} install directory, or `/etc/metricbeat` for rpm and deb.
+
+Disable the `output.elasticsearch` section by commenting it out, then enable
+the `output.logstash` section by uncommenting it: 
+
+[source,yaml]
+----
+#-------------------------- Elasticsearch output ------------------------------
+#output.elasticsearch:
+  # Array of hosts to connect to.
+  #hosts: ["localhost:9200"]
+.
+.
+.
+#----------------------------- Logstash output --------------------------------
+output.logstash:
+  # The Logstash hosts
+  hosts: ["localhost:5044"]
+----
+
+
+Save the file, then restart {metricbeat} to apply the changes.
+
+
+[float]
+[[logstash-setup]]
+==== Configure {ls} to read from the {beats} input
```

Comment:
I would suggest retitling this section `Configure {ls} to listen for {beats} input`.

When started with this pipeline, Logstash will passively listen on the configured port for new connections, but the "read from" phrasing makes it sound like Logstash is actively finding the beats and pulling data from them, which could lead to misconceptions and confusion about how the two work together.